client/blocks/reader-share/index.jsx

```diff
@@ -117,7 +119,14 @@ class ReaderShare extends Component {
 							iconSize: this.props.iconSize,
 						} )
 					) : (
-						<Gridicon icon="reblog" size={ this.props.iconSize } />
+						<>
+							<Gridicon icon="reblog" size={ this.props.iconSize } title={ reblogTitle } />
```

Comment:
I think at some point we'll want to have this controlled by the individual reblog preferences https://github.com/Automattic/wp-calypso/issues/57068

Reply:
Interesting! 🤔 Im not sure if we do want that or not. Those settings make sense for the site front-end itself, but the reader is essentially its own site that should show those buttons its own way. it might look pretty weird if every other postcard in the reader has a different version of the reblog button.  Regarding other settings there, Grant makes some great arguments towards the bottom of that issue.